api/src/test/java/org/projectnessie/tools/catalog/migration/api/CatalogMigrationUtilTest.java

```diff
@@ -129,6 +130,7 @@ public void testBuildNessieCatalog() throws Exception {
     properties.put("warehouse", tempDir.toAbsolutePath().toString());
     properties.put("ref", "main");
     properties.put("uri", "http://localhost:19120/api/v1");
+    properties.put("enable-api-compatibility-check", "false");
```

Comment:
Iceberg 1.3.0 uses Nessie 0.59.0 client. Which has compatibility check enabled and tries to connect with Nessie server (even though it is not up) during init phase itself. 

This is fixed in 0.60.1 but we need a new Iceberg release for that. Hence, disable the compatibility check.

Reply:
just wondering: can you link which fix you are referring to? could it be that compat check only works with v2 api endpoint? maybe we should try to adjust the uri above